src/sage/stats/distributions/discrete_gaussian_lattice.py

```diff
@@ -458,16 +809,16 @@ def _call(self):
 
         EXAMPLES::
 
-            sage: from sage.stats.distributions.discrete_gaussian_lattice import DiscreteGaussianDistributionLatticeSampler
-            sage: D = DiscreteGaussianDistributionLatticeSampler(ZZ^3, 3.0, c=(1/2,0,0))
-            sage: L = [D._call() for _ in range(2^12)]  # long time
-            sage: mean_L = sum(L) / len(L)              # long time
-            sage: norm(mean_L.n() - D.c) < 0.25         # long time
+            sage: D = distributions.DiscreteGaussianDistributionLatticeSampler(ZZ^3, 3.0, c=(1/2,0,0))
+            sage: L = [D._call() for _ in range(2^12)]
+            sage: mean_L = sum(L) / len(L)
+            sage: norm(mean_L.n() - D.c()) < 0.25
             True
 
         .. note::
```

Comment:
```suggestion
        .. NOTE::
```
A while-we-are-at-it thing.